packages/react-app/src/pages/edit.tsx

```diff
@@ -599,14 +604,25 @@ const EditGiveaway = () => {
               >
                 Cancel
               </Button>
-              <Button
+              <LoadingButton
```

Comment:
We should probably also prevent the user from clicking the cancel button, since it will not prevent the operation from completion